proxy-to-sidecar/proxy-to-sidecar.go

```diff
@@ -42,6 +53,14 @@ func Handler(context *nuclio.Context, event nuclio.Event) (interface{}, error) {
 func InitContext(context *nuclio.Context) error {
 	sidecarHost := os.Getenv("SIDECAR_HOST")
 	sidecarPort := os.Getenv("SIDECAR_PORT")
+
+	// enrich sidecar host
+	if sidecarHost == "" {
+		sidecarHost = "http://localhost"
```

Comment:
not sure if it's needed since it's a template and we have default value the host -  [  defaultValue: "http://0.0.0.0"](https://github.com/nuclio/nuclio-templates/blob/d55a74d60040273022eacfaae7c1986aaa76e638/proxy-to-sidecar/function.yaml.values#L6)